src/bp/ui-shared/src/translations/fr.json

```diff
@@ -55,12 +56,18 @@
   "delete": "Supprimer",
   "deleteCondition": "Supprimer la condition",
   "deleteContent": "Supprimer le contenu",
+  "deleteField": "Supprimer le champ",
   "deleteNode": "Supprimer le bloque",
   "deletePrompt": "Supprimer le prompt",
+  "deleteVariable": "Supprimer la variable",
+  "deleteVariableFromLibrary": "Supprimer la variable de la librairie",
+  "deleteWorkflow": "Supprimer le workflow",
   "description": "Description",
   "details": "Détails",
   "disabled": "Désactivé",
   "duplicate": "Dupliquer",
+  "duplicateVariableType": "Dupliquer le type de variable",
+  "duplicateWorkflow": "Dupliquer le workflow",
```

Comment:
These translations are not single words or reusable, please put them in the studio